github.go

```diff
@@ -185,9 +193,68 @@ func (a *App) GetLatestReleaseData(repo string) (*GhRelease, error) {
 	return &a.latest_release_json[len(a.latest_release_json)-1].content, nil
 }
 
-// func (a *App) CheckForNewRelease(tag string) (bool, error) {
-// 	//go:build !production
-// 	return false, nil
+func (a *App) UpdateBotpack(repo string, installPath string, currentTag string) (string, error) {
+	latest_release, err := a.GetLatestReleaseData(repo)
+	if err != nil {
+		return "", err
+	}
+
+	var file_name string
+	if runtime.GOOS == "windows" {
```

Comment:
todo: maybe check platform too?
we should add a todo comment at least

Reply:
i added the todo comment because the botpack currently is only built for x64